README-it.md

```diff
@@ -0,0 +1,82 @@
+
+## Contribuire ai documenti
+
+Puoi fare clic sul pulsante **Fork** nell'area in alto a destra dello schermo per creare una copia di questo repository nel tuo account GitHub. Questa copia è chiamata * fork *. Apporta le modifiche desiderate nella tua fork e quando sei pronto a inviarci tali modifiche, vai alla tua fork e crea una nuova richiesta di pull per farcelo sapere.
+
+Una volta creata la richiesta di pull, un revisore di Kubernetes si assumerà la responsabilità di fornire un feedback chiaro e attuabile. Come proprietario della richiesta pull, ** è tua responsabilità modificare la tua richiesta di pull per rispondere al feedback che ti è stato fornito dal revisore di Kubernetes. ** Nota anche che potresti finire con più di un revisore di Kubernetes feedback o si potrebbe finire per ricevere feedback da un revisore di Kubernetes diverso da quello originariamente assegnato per fornire un feedback. Inoltre, in alcuni casi, uno dei revisori potrebbe chiedere una revisione tecnica da un [revisore tecnico di Kubernetes] (https://github.com/kubernetes/website/wiki/Tech-reviewers) quando necessario. I revisori faranno del loro meglio per fornire feedback in modo tempestivo, ma i tempi di risposta possono variare in base alle circostanze.
+
+Per ulteriori informazioni su come contribuire alla documentazione di Kubernetes, vedere:
+
+* [Inizia a contribuire](https://kubernetes.io/docs/contribute/start/)
+* [Staging Your Documentation Changes](http://kubernetes.io/docs/contribute/intermediate#view-your-changes-locally)
+* [Uso dei modelli di pagina](http://kubernetes.io/docs/contribute/style/page-templates/)
+* [Guida allo stile della documentazione](http://kubernetes.io/docs/contribute/style/style-guide/)
+* [Localizzazione della documentazione di Kubernetes](https://kubernetes.io/docs/contribute/localization/)
+
+## La documentazione di localizzazione di Kubernetes di `README.md`
+
+### Coreano
+
+Vedi la traduzione di `README.md` e una guida più dettagliata per i contributori coreani nella pagina [Korean README](README-ko.md). 
+
+Puoi raggiungere i manutentori della localizzazione coreana a:
+
+* June Yi ([GitHub - @gochist](https://github.com/gochist))
+* [Slack channel](https://kubernetes.slack.com/messages/kubernetes-docs-ko)
+
+## Eseguire il sito localmente usando Docker
+
+Il modo consigliato per eseguire localmente il sito Web di Kubernetes è eseguire un'immagine specializzata [Docker](https://docker.com) che include il generatore di siti statici [Hugo](https://gohugo.io).
+
+> Se si utilizza Windows, sono necessari alcuni altri strumenti che è possibile installare con [Chocolatey](https://chocolatey.org). `choco install make`
+
+> Se preferisci eseguire il sito Web localmente senza Docker, vedi [Esegui localmente il sito usando Hugo](# running-the-site-local-using-hugo) di seguito.
+
+Se hai Docker [attivo e funzionante](https://www.docker.com/get-started), crea l'immagine del Docker `kubernetes-hugo` localmente:
+
+```Bash
```

Comment:
Lowercase
```suggestion
```bash
```

Reply:
done

Reply:
I couldn't find any new commit and the first commit is still containing the uppercase variant, therefore I'm 'unresolving' these ones.